### PR TITLE
Sample course: add sample comments to 'First Team Feedback Session' #7493

### LINKED ISSUE #7493: Sample course: add sample comments to 'First Team Feedback Session'
Sample comments for feedback responses in First Team Feedback Session.

### PR DESCRIPTION
**Pull request by Mynk96**
Fixes #7493 
2 in 'comment2FromT*C*ToR*Q*S*C*' indicates comments for 'first team feedback session'.

### CODE REVIEW

**Comment by Mynk96** [comment]
Ready for review!

**Comment by damithc** [comment]
> Looking at this new set of changes, I suspect my original guess for the meaning of S was wrong - perhaps it should have been for "session" instead of "section".

We can consider updating all names to a clearer naming convention :-)

**Comment by Mynk96** [comment]
@chowyb Can we have the convention like this?
`"FRC1FromI1COU1ToFS1Q2R6":`
where :
FRC1 - stands for feedbackResponseComment number 1,
I1 - Instructor 1,
COU1 - course number 1,
FS *- Feedback Session
(1 -> `Session with different question types`
2 -> `First team feedback session`
3 -> `Second team feedback session`)
Also, FS2 has been used to describe `First team feedback session` previously in the file.
Q* - question number
R* - Response number for that question

**Comment by chowyb** [comment]
@Mynk96 that could work. However, if you plan to update the names, it should be done in a separate issue, since it'll have to touch the .json files used in several other tests too.

**Comment by Mynk96** [comment]
@chowyb Got it. So will stick to the current convention in this PR, after completion will create a new PR to improve the naming convention.

**Comment by Mynk96** [comment]
@chowyb Ready for Review!

File: src/main/resources/InstructorSampleData.json [hunk 1]
```diff
@@ -2868,6 +2868,126 @@
...
+        "GIVER",
+        "RECEIVER",
+        "RECEIVER_TEAM_MEMBERS",
+        "OWN_TEAM_MEMBERS",
+        "INSTRUCTORS"
+      ],
+      "isVisibilityFollowingFeedbackQuestion": false,
+      "createdAt": "2012-04-30 9:36 PM UTC",
+      "commentText": {
+        "value": "<p>Well, Being your first team project always takes some time.I hope you had nice experience working with the team.</p>"
```
Comment (damithc) [inline src/main/resources/InstructorSampleData.json]: `time.I` -> `time. I`?

**Comment by chowyb** [comment]
> Also, how come 4 comments are added but only 2 show up in the tests?

The HTML file that was changed is used for testing the feedback results, as seen by the student associated with the new instructor account, which is in "Tutorial Group 2". Therefore, only the two comments added to responses under "Tutorial Group 2" is visible on that page.

The test that was intended to show the feedback results as seen by the instructor may not have auto-loaded the answers correctly, resulting in a default "no responses" message for each of the questions, and therefore also excluding feedback response comments as a result. This is currently being addressed in #7511.

**Comment by damithc** [comment]
Can merge after CI passes

File: src/test/resources/pages/newlyJoinedInstructorStudentFeedbackResultsPage.html [hunk 2]
```diff
@@ -269,6 +269,24 @@ <h4 id="interpret">
...
+                <ul class="list-group comment-list">
+                  <li class="list-group-item list-group-item-warning" id="responseCommentRow-${comment.id}">
+                    <div id="commentBar-${comment.id}">
+                      <span class="text-muted">
+                        From: AHPUiT+++_.instr1!@gmail.tmt [Mon, 30 Apr 2012, 09:36 PM UTC]
+                      </span>
+                    </div>
+                    <div id="plainCommentText-${comment.id}" style="margin-left: 15px;">
+                      <p>
+                        Well, Being your first team project always takes some time.I hope you had nice experience working with the team.
```
Comment (whipermr5) [inline src/test/resources/pages/newlyJoinedInstructorStudentFeedbackResultsPage.html]: There's a missing space here causing the build failure. Was this updated with GodMode?

**Pull request merged**